src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -465,6 +465,117 @@ paths:
         connectionId: '${vpc_connection_id}'
         httpMethod: POST
 
+
+  #
+  # Anonymous Elector Document Applications
+  # --------------------------------------------------------------------------------
+  '/eros/{eroId}/anonymous-applications':
```

Comment:
I've since seen the response object at the bottom of this PR and have suggested `AedSearchSummary` as an alternative. Perhaps therefore this endpoint could be `/eros/{eroId}/aed-search-summaries` which would mean we wouldn't need to change `/eros/{eroId}/anonymous-elector-documents`.

However, I'm not particularly enamoured by that idea and it's making me wonder if "summary" is in danger of becoming an overloaded term? (though not as bad as `PENDING`! 😆). I guess the question is what are these response objects and when does an object stop being the "full" resource and start being a "summary"? 

I'm not saying any of this is easy btw! 😆

Reply:
For the same reason that you mentioned with existing get anonymous-elector-documents endpoint , i named this to anonymous-applications. This name is similar to how we search for anonymous-applications within vca

Reply:
Could rename exisitng get summaries endpoint to be "history" or something along those lines if UI isnt consuming those.

The current pr is more of "anonymous-elector-documents" then

Reply:
It's just occurred to me that the existing GET endpoint could have `/{applicationId}` in the path rather than the body, which would make sense conceptually.

Reply:
Endpoint and tag renamed to `...../anonymous-elector-document-summaries` @matt-wills-valtech